index.js

```diff
@@ -20,9 +20,48 @@ function buildRequestor(clientOptions) {
 };
 
 function buildLogger(clientOptions) {
-  return clientOptions.logger
-    || (clientOptions.logContainer && clientOptions.logContainer.get('smartsheet'))
-    || winston.loggers.get('smartsheet');
+  if((clientOptions.logger && clientOptions.loggerContainer)
+      || (clientOptions.logger && clientOptions.logLevel)
+      || (clientOptions.loggerContainer && clientOptions.logLevel)) {
```

Comment:
Can you extract this boolean logic into a helper method?

Reply:
👍